free-coffee-cups/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+<script>
+// if less than 6 return the value
+//else divide coffeecups by 6 and store that value into a variable
+//add that value to freecoffeecups and return
+
+  function freeCoffeeCups(coffeeCups){
+    if (coffeeCups<6){
```

Comment:
I'm not sure why you isolated this case, the code you wrote in your else statement here should work for this condition as well 🤔